proposals/0007-access-policy.md

```diff
@@ -0,0 +1,126 @@
+---
+simd: '0007'
+title: SIMD Access Policy
+authors:
+- Jacob Creech (Solana Foundation)
+category: Meta
+type: Meta
+status: Draft
+created: 2022-12-10
+---
+
+## Summary
+
+3 levels of access, in the order of increasing access:
+
+1. Triage
+
+    Requirement: One voucher from anyone with level 2 or above access.
+
+2. Core
```

Comment:
Consider renaming "Core" to "Write"? Now we essentially have something that directly maps into the standard github repository roles

Reply:
Renamed. Originally wanted to make it obvious that "Core Contributors" had "Core Access", but the longer I thought about it the name only caused more confusion.